services/contentapi/main.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+app = FastAPI(
+    title="RAISE Content API"
+)
+
+
+class ContentData(BaseModel):
+    id: str
+    content: str
+
+
+@app.get("/contents/{content_id}", response_model=ContentData, )
+async def create_event(content_id):
+
+    lookup = {
+              "video_1":
+              "<div><p>This is a video for ID {content_id}</p><div>",
+              "recording_1":
+              "<div><p>This is a recording for ID {content_id}</p><div>"
+              }
+
+    if content_id in lookup:
+        data = ContentData(id=content_id, content=lookup[content_id])
+    else:
+        data = "No content for that content_id"
+    return data
```

Comment:
I think for the purposes of the `k12-64` spike work we can just assume every `content_id` is valid and just return something like the following for `content`:

```python
content = f""<div><p>This is content for ID {content_id}</p><div>""

return ContentData(id=content_id, content=content)
```

That way `GET` requests for things like `/contents/1` , `/contents/2`, `/contents/44596cea-0bf7-4769-9d94-fac4e74e1639` all return unique data so we can differentiate when rendering. While error handling of invalid data is something good to think about, since this spike is about technical viability I think we can focus on the "happy path" and deal with `404`s, timeouts, etc. in the client side code for a "production" filter plugin implementation if things work out.